ml_packages_in_R.rmd

```diff
@@ -0,0 +1,123 @@
+#Low-key ML packages in R
+#Author - Vedant Kumar and Siddhant Kumar
+
+#Part 1 - Nuralnet Package in R
+
+```{r}
+# install package
+install.packages("neuralnet")
+library("neuralnet")
+```
+
+```{r}
+# creating training data set
+weight=c(120,80,55,100,75,60)
+height=c(150,165,155,175,145,170)
+obese=c(1,0,0,1,1,0)
+
+df=data.frame(weight,height,obese)
+```
+
+```{r}
+# fit neural network
+nn=neuralnet(obese~weight + height,data=df, hidden=3,act.fct = "logistic",
+             linear.output = FALSE)
+
+# - obese~weight + height, Placed is label(dependent variable) and 
+#weight and height are features (independent variable).
+# - df is dataframe,
+# - hidden = 3: represents single layer with 3 neurons.
+# - act.fct = "logistic" is the activation function
+```
+
+```{r}
+# plot neural network
+plot(nn)
+```
+
+```{r}
+#prepare a test data
+weight=c(130,90)
+height=c(130,185)
+df_test=data.frame(weight,height)
+```
+
+```{r}
+## Prediction using neural network
+Predict=compute(nn,df_test)
+prob = Predict$net.result
+pred = ifelse(prob>0.5, 1, 0)
+print(pred)
+```
+
+#Part 1 - quantmod Package in R
+
+```{r}
+#install packages
+install.packages("quantmod")
+library(quantmod) 
```

Comment:
Include any library() statements at the top of your .Rmd file, below the title, name, and setup, but above any content.